examples/benchmark/profet_benchmark.py

```diff
@@ -0,0 +1,136 @@
+""" Example script for using tasks from the Profet algorithm.
+
+This can be called like so:
+```console
+python examples/benchmark/profet_benchmark.py --task_type svm
+```
+"""
+import logging
+from dataclasses import dataclass
+from logging import getLogger as get_logger
+from pathlib import Path
+from typing import List, Type
+from orion.algo.base import OptimizationAlgorithm
+
+from orion.benchmark.assessment import AverageResult
+from orion.benchmark.benchmark_client import get_or_create_benchmark
+from orion.benchmark.task.profet import FcNetTask, ForresterTask, SvmTask, XgBoostTask
+from orion.benchmark.task.profet.profet_task import MetaModelTrainingConfig, ProfetTask
+
+try:
+    from simple_parsing.helpers import choice
+except ImportError as exc:
+    raise RuntimeError(
+        "Need simple-parsing to be installed to run this example.\n"
+        "You can install it using `pip install simple-parsing`."
+    ) from exc
+
+logger = get_logger("orion")
+logger.setLevel(logging.DEBUG)
+
+from orion.benchmark.task.profet import FcNetTask, ForresterTask, SvmTask, XgBoostTask
+
+algos_available = set(OptimizationAlgorithm.types.keys()) - {"primaryalgo"}
+
+
+@dataclass
+class ProfetExperimentConfig:
+    """ Configuration option for the demo of the Profet tasks. """
+
+    # The type of Profet task to create.
+    task_type: Type[BaseTask] = choice(  # type: ignore
+        {"svm": SvmTask, "fcnet": FcNetTask, "xgboost": XgBoostTask, "forrester": ForresterTask}
+    )
+
+    # Name of the experiment.
+    name: str = "profet"
+    # Configuration options for the training of the meta-model used in the Profet tasks.
+    profet_train_config: MetaModelTrainingConfig = MetaModelTrainingConfig()
+
+    algorithms: List[str] = choice(*algos_available, default_factory=["random", "tpe"].copy)
+
+    # Number of repetitions for each experiment
+    n_repetitions: int = 10
+    # Optimization budget (max number of trials) for optimizing the task.
+    max_trials: int = 50
+    # Run in debug mode:
+    # - No presistent storage
+    # - More verbose logging
+    # (@TODO: This isn't technically correct: Benchmarks don't support the `debug` flag in
```

Comment:
Is there something to do here? Changing the value or removing the argument later on?

Reply:
Nothing to do here, but later on when the other PR goes through we'll be able to pass the `debug` argument to the Benchmark constructor.

Reply:
Ok, so TODO note could be removed?

Reply:
No sorry, that wasn't clear: We'll have to change a line below so we actually pass the `debug` argument to the `Benchmark` constructor, which doesn't currently accept it.

Reply:
Oh ok! You can add the URL to that other PR so that we do not loose trace. Or an issue may be better.

Reply:
https://github.com/Epistimio/orion/issues/674